.pre-commit-config.yaml

```diff
@@ -0,0 +1,4 @@
+# This is the configuration for pre-commit, a local framework for managing pre-commit hooks
+#   Check out the docs at: https://pre-commit.com/
+
+repos: []
```

Comment:
`pre-commit` will add a newline here at the end of the file :) Not important right now.